helm/macropower-analytics-panel-server/templates/deployment.yaml

```diff
@@ -64,10 +64,25 @@ spec:
           limits:
             cpu: 250m
             memory: 250Mi
+        {{- if not .Values.letsencrypt }}
+        volumeMounts:
+        - name: trusted-ca
+          mountPath: /etc/ssl/certs
```

Comment:
I think that's what causes the issue with Grafana not being able to talk to any ssl endpoint.

`/etc/ssl/certs` is system certificates are installed we can't just override this folder it's essential for ssl connections to work!